packages/apollo/src/load-schema.ts

```diff
@@ -7,28 +7,15 @@ export async function loadSchema(
   config: ApolloConfig
 ): Promise<GraphQLSchema | undefined> {
   if (dependency.schema) {
-    try {
-      return await fetchSchema(
-        { url: dependency.schema },
-        config.projectFolder
-      );
-    } catch {}
```

Comment:
Throwing away all errors which occur from the `fetchSchema` method buries not only just the mere failure to acquire the schema in _some_ manner, but it also neglects to reveal very important information about HTTP transport errors, file system errors, GraphQL execution and more — all of which happen via `fetchSchema`!

Reply:
Faced this yesterday, because of this catch - SSL certificate error was swallowed.